template/{{.input_root_dir}}/.github/workflows/{{.input_project_name}}-bundle-cd-staging.yml.tmpl

```diff
@@ -0,0 +1,40 @@
+# This GitHub workflow deploys Bundle resources (ML resource config and more)
+# defined under {{template `project_name_alphanumeric_underscore` .}}/databricks-resources/*
+# and {{template `project_name_alphanumeric_underscore` .}}/bundle.yml with staging environment configs,
+# when PRs are merged into the default branch
+name: Bundle Deployment for {{template `project_name` .}} Staging
+
+on:
+  push:
+    branches:
+      - '{{template `default_branch` .}}'
```

Comment:
wondering what is `.` for, is it part of the template syntax?

Reply:
Yes it's required by templates.
The element after the function name string is the input parameter. `.` is the current scope object.